suite/src/models/modelConstants.ts

```diff
@@ -1,6 +1,6 @@
 import { type ModelDefinition } from '@ceramicnetwork/stream-model'
 
-export const newModel: ModelDefinition = {
+export const MODEL_DEFINITION_LIST: ModelDefinition = {
```

Comment:
nitpick, can we call this LIST_MODEL_DEFINITION or MODEL_LIST_DEFINITION (and apply the same convention to the other constants here).

When I first saw this I thought it was a list of model definitions, not a definition of a model with the list account relation.